anaconda.py

```diff
@@ -227,7 +227,9 @@ def setup_environment():
         sys.exit(0)
 
     log.info("%s %s", sys.argv[0], util.get_anaconda_version_string(build_time_version=True))
-    log.debug("Image packages list: %s", util.get_image_packages_info())
+    # Do not exceed default 8K limit on message length in rsyslog
+    for log_line in enumerate(util.get_image_packages_info(max_string_chars=8096-100)):
+        log.debug("Image packages: %s", log_line)
```

Comment:
I think you don't need the enumerate?

Reply:
Ah I don't, it is a leftover... I was thinking I would enumerate the log messages but then realized it isn't work it.

Reply:
Removed